playwright.config.ts

```diff
@@ -59,6 +59,10 @@ const config: PlaywrightTestConfig = {
       timeout: 120000,
       use: {
         ...devices["Desktop Chrome"],
+        viewport: {
+          width: 1920,
+          height: 1280,
+        },
```

Comment:
Why this change was needed?

Reply:
Suspected that viewport size was the reason the end to end tests were failing which turned out to be wrong, it's back to normal now.